cache/remotecache/export.go

```diff
@@ -185,6 +186,11 @@ func (ce *contentCacheExporter) Finalize(ctx context.Context) (map[string]string
 		return nil, err
 	}
 
+	if len(config.Layers) == 0 {
+		bklog.G(ctx).Warn("failed to match any cache with layers")
```

Comment:
I trivially copied this warning from the equivalent code in the `inline` exporter. It's not super-expressive, which suggests no one really cares about the warning, or is even looking for it, in the `inline` exporter. (Unless `bklog.G(ctx)` is doing more than I think it is)

If we really do want a warning or similar, it might make sense to define an error type for `ErrNoCacheLayersToExport` or something, and log that occurance with relevant details at a higher level (common to all exporters) and/or fail if that is an unexpected/undesirable case by that logic.

Reply:
I think we should also do a `progress.OneOff` call in here with "skipping cache export for empty result" message so the user sees what is happening on the progress output.

Reply:
I guess this is what you intended? The API's a little weird for this use-case, perhaps this function should have a single progress writer it uses rather than a bunch of `OneOff`s?